manifests/h/hp/support/assistant/12.19.53.13/hp.support.assistant.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: hp.support.assistant
+PackageVersion: 12.19.53.13
+PackageLocale: en-US
+Publisher: HP Computers
+PackageName: HP Support Assistant
+License: Copyright c 2020-2021 Hewlett Packard
+ShortDescription: HP Support Assistant

```

Comment:
ShortDescription: HP Support Assistant is a free utility on Hewlett Packard computers released after 2012 that helps avoid and resolve problems with the computer using updates and self-help options.

Reply:
Path names should be capitalized as well  `HP/Support/Assistant`

Reply:
@mddtest thank you for helping improve PRs.

Did you know you can use the "[suggest](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)" syntax to make it easier for the author to just hit "accept changes"